packages/cactus-cmd-api-server/src/main/typescript/config/config-service.ts

```diff
@@ -572,7 +571,7 @@ export class ConfigService {
       cockpitWwwRoot: (schema.cockpitWwwRoot as SchemaObj).default,
       cockpitCorsDomainCsv: (schema.cockpitCorsDomainCsv as SchemaObj).default,
       cockpitMtlsEnabled: false,
-      cockpitTlsEnabled,
+      cockpitTlsEnabled: false,
```

Comment:
@AzaharaC TLS enablement needs to come from the secure defaults by default (one of our design principles)